coalib/coala.py

```diff
@@ -71,6 +71,9 @@ def main():
     except BaseException as exception:  # pylint: disable=broad-except
         return get_exitcode(exception, log_printer)
 
+    if args.non_interactive and args.format:
+        return mode_format()
+
     if args.non_interactive:
```

Comment:
Shouldn't this be `elif args.non_interactive:` and the statement on L80 be `elif args.format:` ?
We're returning after checking for format/ci so if `args.non_interactive==true` we don't need to check if `args.format==true` since the case for both being true is handled by L74.